lib/Common/DataStructures/Comparer.h

```diff
@@ -155,9 +155,10 @@ struct StringComparer
         return hash;
     }
 
-    static int Compare(T str1, T str2)
+    inline static int Compare(T str1, T str2)
     {
-        return ::wcscmp(str1, str2);
+        const int result = ::wcscmp(str1, str2);
```

Comment:
I had mentioned this one on the details of this PR. I couldn't find any consumer for that interface yet still normalized the output from libc call to prevent future consistency